docs/release-notes/Percona-Everest-1.2.0-(2024-09-09).md

```diff
@@ -0,0 +1,204 @@
+
+- [EVEREST-1212](https://perconadev.atlassian.net/browse/EVEREST-1212) - Starting with Percona Everest 1.2.0, you can now directly edit the monitoring endpoint from the database overview page, instead of having to use the Edit database wizard.
+
+
+- [EVEREST-1230](https://perconadev.atlassian.net/browse/EVEREST-1230): We've updated the **Resources** panel on the **Database overview** page to be independent instead of part of the **DB Details** panel and improved the overall look and feel of this page.
+
+
+- [EVEREST-1369](https://perconadev.atlassian.net/browse/EVEREST-1369): We've updated the documentation for the section on turning off telemetry.
+
+
+## The latest in APIs: What’s new and what’s deprecated
+
+
+=== "Newly added API endpoints"
+
+    Check out the new API endpoints we've added in Percona Everest 1.2.0:
+
+    |**No**|**API endpoints**|**Method**|
+    |---------|---------------------|---------|
+    |**1.**|`/namespaces/{namespace}/monitoring-instances`|a.`GET`</br> b.`POST`|
+    |**2.**|`/namespaces/{namespace}/monitoring-instances/{name}`|a.`GET`</br>b. `PATCH`</br>c.`DELETE`|
+    |**3.**|`/namespaces/{namespace}/backup-storages`|a.`GET`</br> b. `POST`|
+    |**4.**|`/namespaces/{namespace}/backup-storages/{name}`|a.`GET`</br> b.`POST`|
+    |**5.**|``/permissions``|a.`GET`|
+
+
+
+=== "Deprecated API endpoints"
+
+    This is the list of the API endpoints deprecated from Percona Everest:
+
+    |**No**|**API endpoints**|**Method**|
+    |---------|---------------------|---------|
+    |**1.** |**Endpoints deprecated from Percona Everest v1.1.0:**|
+    |**a.**|`/namespaces/{namespace}/database-engines/{name}/operator-version/preflight`|1.`GET`|
+    |**b.**|`/namespaces/{namespace}/database-engines/{name}/operator-version`|1.`GET`</br>2.`PUT`|
+    |**2.** |**Endpoints deprecated from Percona Everest v1.2.0:**|
+    |**c.**|`/monitoring-instances`|1.`GET`</br> 2.`POST`|
+    |**d.**|`/monitoring-instances/{name}`|1.`GET`</br>2. `PATCH`</br>3.`DELETE`|
+    |**e.**|`/backup-storages`|1.`GET`</br> 2.`POST`|
+    |**f.**|`/backup-storages/{name}`|1.`GET`</br>2. `PATCH`</br>3.`DELETE`|
+
+## Bugs
+
+- [EVEREST-768](https://perconadev.atlassian.net/browse/EVEREST-768): The PostgreSQL pods now demonstrate the intended behavior by not getting stuck, as they automatically restart when the database is restarted.
+
+
+- [EVEREST-1232](https://perconadev.atlassian.net/browse/EVEREST-1232): The backups and restore pages have been updated to show consistent date formats.
+
+- [EVEREST-1253](https://perconadev.atlassian.net/browse/EVEREST-1253): The **Delete** option in the backup menu is properly now disabled if the backup is in the **Deleting** status.
+
+- [EVEREST-1273](https://perconadev.atlassian.net/browse/EVEREST-1273) Backend sends an empty response if the backup storage URL is edited to an HTTP address.
```

Comment:
The fix is that now we show an error message if the user edits the backup storage URL to an invalid one.